kubernetes/deployments/frontend.yaml

```diff
@@ -1,9 +1,12 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: frontend
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: frontend
```

Comment:
recommend also adding "track: stable" to selector.matchLabels to match both labels in the template